frontend/src/concepts/pipelines/content/tables/experiment/ExperimentTableRow.tsx

```diff
@@ -16,21 +21,29 @@ const ExperimentTableRow: React.FC<ExperimentTableRowProps> = ({
   onToggleCheck,
   experiment,
   actionColumnItems,
-}) => (
-  <Tr>
-    <CheckboxTd id={experiment.experiment_id} isChecked={isChecked} onToggle={onToggleCheck} />
-    <Td dataLabel="Experiment">{experiment.display_name}</Td>
-    <Td dataLabel="Description">{experiment.description}</Td>
-    <Td dataLabel="Created">
-      <ExperimentCreated experiment={experiment} />
-    </Td>
-    <Td dataLabel="Last 5 runs">
-      <LastExperimentRuns experiment={experiment} />
-    </Td>
-    <Td isActionCell dataLabel="Kebab">
-      <ActionsColumn items={actionColumnItems} />
-    </Td>
-  </Tr>
-);
+}) => {
+  const { namespace } = usePipelinesAPI();
+
+  return (
+    <Tr>
+      <CheckboxTd id={experiment.experiment_id} isChecked={isChecked} onToggle={onToggleCheck} />
+      <Td dataLabel="Experiment">
+        <Link to={experimentRunsRoute(namespace, experiment.experiment_id)} state={{ experiment }}>
```

Comment:
This is the whole experiment object and the route is only the experiment ID.

Reply:
Yeah, it's snappier if we pass both... route is needed cuz that's where the page is at, the object is useful as it can be quick to load the page if we provide known data over